content/vendor/guides/ha-guide.md

```diff
@@ -0,0 +1,411 @@
+
+![Add-License](/images/guides/kots/ha-cluster-add-license.png)
+
+Once you upload your license it may take a few minutes to load and continue with the deployment.
+
+
+![License-Load](/images/guides/kots/ha-cluster-license-load.png =250x250)
+
+Once the license is loaded, the next window will depend on the application being deployed as this window renders whatever has been codified in the application's [Config.yaml](https://kots.io/reference/v1beta1/config/).
+
+After the configuration window, you should see the preflight checks and eventually land on a window similar to what is shown below
+
+![KotsAdmin](/images/guides/kots/ha-cluster-app-dployed.png)
+
+
+### Verify the Deployment
+
+To see everything that has been deployed, ssh into one of the nodes and run:
+
+```shell
+kubectl get pods --all-namespaces -o wide
+```
+
+
+The output will show you all the pods running on all namespaces and on which node. Since we only have a one node cluster, all pods should be running on node 1. Check the status of the pods to make sure there are no issues.
+
+![AllPods](/images/guides/kots/ha-cluster-all-pods-one-node.png)
+
+If you are following along, you should be able to access the application by browsing to http://<ip-address-of-ha-proxy>. The sample app uses flask 'routes' to call various methods to interact with Postgres. To test if it can write to the database, use the  `/sql-check`   route which will check connection to the default database and list the databases are avaialble to the 'postgres' user. To write to the database, first use the `/sql-create` route which creates a database (appdiretdb) with a table (tblrecords) in it. Finally, use the `/sql-add` route to add a row the table with the timestamp. Once the database and table are created, simply use this route to add more records to the database.
+
+To verify that the data is in fact being written to Postgres, install pgAdmin and create a connection to the database using the Load Balancer's IP address. Once conneted, you should see the database and table. To retrieve records you can run a new query like this:
+
+`SELECT * FROM tblrecords`
+
+You should see records for each time that the /sql-add route is accessed. Below is a screenshot of what it should look like:
+
+![pgAdmin](/images/guides/kots/ha-cluster-pg-admin.png)
+
+### Adding Remaining Nodes to Cluster
+
+Under Cluster Management you can view the status of the embedded cluster. At this time, it only has one node. 
+
+
+![ClusterManagement](/images/guides/kots/ha-cluster-cluster-mgmt.png)
+
+To get the commands to run on the other nodes, click on the 'Add Node' button. From here you can retrieve the command to use on Worker and Master nodes. For our exercise we will add Master nodes to give us a proper HA solution.
```

Comment:
I think "HA Cluster" makes more sense than "HA solution" here, since the cluster is only a piece of the solution, but this one is super subjective so it's your call.

```suggestion
To get the commands to run on the other nodes, click on the 'Add Node' button. From here you can retrieve the command to use on Worker and Master nodes. For our exercise we will add Master nodes to give us a proper HA cluster.
```